src/DevHome.csproj

```diff
@@ -85,6 +86,9 @@
 
   <ItemGroup>
     <ProjectReference Include="..\common\DevHome.Common.csproj" />
+    <ProjectReference Include="..\service\Devhome.Service.IDL\Devhome.Service.IDL.vcxproj" />

```

Comment:
Can we add these to the project architecture mermaid diagram in architecture.md?

Reply:
Done.